1-JavaScript/Week3/homework/ex3-lemonAllergy.test.js

```diff
@@ -26,26 +10,27 @@ const fruitBasket = [
 ];
 
 // ! Function under test
-function sanitizeFruitBasket(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function sanitizeFruitBasket(fruitsOfBasket , fruitsToRemove) {
+const unwantedFruits = fruitsOfBasket.filter( fruit => fruit !==fruitsToRemove);
+return unwantedFruits;
```

Comment:
Nice 👍 I like the implicit return you are using here, very clear. One thing: it's a bit confusing that the name is `fruitsToRemove`, this makes it seem like you can pass multiple fruits but it's only one in this implementation.

Also, should the name be `unwantedFruits`? 😅 You want to return a fruit basket with all the fruits that are not equal to `fruitToRemove`, so a more fitting name can be `wantedFruits`?

```suggestion
function sanitizeFruitBasket(fruitsOfBasket , fruitToRemove) {
const unwantedFruits = fruitsOfBasket.filter(fruit => fruit !== fruitToRemove);
return unwantedFruits;
```